Resources/public/templates/navigation/item.hbt

```diff
@@ -1 +1 @@
-<a href="{{ path route.name route.params }}" class="ez-navigation-item">{{ title }}</a>
+<a href="{{ path route.name route.params }}" class="ez-navigation-item" data-navigation="{{ title }}">{{ title }}</a>
```

Comment:
@dpobel okay, I have already done this, should I include it in this PR or in a separate one?